### PR TITLE
Add fields for To and From and add fields to all route presets.

### PR DESCRIPTION
**Pull request by castriganoj**
I took a stab at this as a first time iD contributor. Please review and let me know if anything if changes are needed. 

Thanks for participating in Hacktoberfest. 

Best, 
Jim

#5408

### CODE REVIEW

**Comment by JamesKingdom** [comment]
@manfredbrandl The changes about Helsingborg and Stockholm don't need to be removed. it's just the imagery information updating when iD has been rebuilt. 
You can see in other PR's such as #5397, that it's just left in.

File: data/presets/fields/to.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "key": "to ",
```
Comment (bhousel) [inline data/presets/fields/to.json]:
```suggestion
    "key": "to",
```

Comment (bhousel) [reply data/presets/fields/to.json]: This has an extra space

**Comment by bhousel** [comment]
> The changes about Helsingborg and Stockholm don't need to be removed. it's just the imagery information updating when iD has been rebuilt.

Agree - this is just different people having different versions of ELI, which we can't help.  It all sorts itself out when I make a release.

**Comment by bhousel** [comment]
Thanks @castriganoj 👍

**Pull request merged**